clustering/config.py

```diff
@@ -0,0 +1,17 @@
+class PreprocessSettings:
+    index_cols = ['PassengerId', 'Name']
+    categorical_cols = ['Sex', 'Embarked']
+    drop_cols = ['Ticket', 'Cabin']
```

Comment:
these are settings for testing, so I suggest moving them to `tests/test_clustering.py`